### ISSUE TITLE
Error thrown when schema has a title or description flag

### ISSUE THREAD

**Issue by Kodiyath** [everit-org/json-schema#263]
The schema considered is 

```
{"title": "feeds","description": "Feeds user subscribes to","type": "object","patternProperties": {"^[A-Za-z]+$": { "type": "boolean" }}}
```

When trying to build the `org.everit.json.schema.Schema` object, the following error is thrown:

```
org.json.JSONException: Duplicate key "title"
	at org.everit.json.schema.internal.JSONWriter.key(JSONWriter.java:261)
	at org.everit.json.schema.internal.JSONPrinter.key(JSONPrinter.java:27)
	at org.everit.json.schema.Schema.lambda$describeTo$0(Schema.java:281)
```
Incase if the title is removed from the schema, the same error is thrown for the field `description`.

I am using the version 1.10.0.

**Comment by afranzi** [comment]
Hi @Kodiyath,

This is related to this issue https://github.com/everit-org/json-schema/issues/250. They fixed in the next release.

**Comment by Kodiyath** [comment]
Thankyou @afranzi, looks like the version is not released yet.

**Comment by afranzi** [comment]
Nop, but you can use the commit hash as @erosb mention in the previous issue, so you can keep up developing without the bug :)

**Comment by erosb** [comment]
Yes, exactly, thanks @afranzi . 
During the week I will try to find time to fix #260 but if I can't make it then I will publish the next release fixing the title/description problem this week.

Thanks both.

**Comment by erosb** [comment]
Hello, I've just released version `1.11.0` with the bugfix.
